articles/cosmos-db/merge.md

```diff
@@ -75,38 +75,25 @@ In PowerShell, when the flag `-WhatIf` is passed in, Azure Cosmos DB will run a
 
 #### [PowerShell](#tab/azure-powershell)
 
+The Azure Cloud Shell is a free interactive shell that you can use to run the steps in this article. It has common Azure tools preinstalled and configured to use with your account. To launch the Cloud Shell, navigate to the Data Explorer in the Azure Portal and click on the icon in the upper right corner next to the search bar. Then, follow the script
```

Comment:
We don't want to infer that the user will use the Azure Cloud Shell. These commands could be ran from their local environment. In a how-to guide or a tutorial, we can recommend the Cloud Shell, but general documentation don't want to make any inferences about their environment.